### ISSUE TITLE
pdtable allows for writing of invalid tables in JSON

### ISSUE THREAD

**Issue by guilhermebs** [startable/pdtable#122]
Code example:

``` python
    pdtable = Table(pd.DataFrame({
        'a': [1, 2, 3],
        'b': ["a", "b", "c"]
    }), name='test', destinations='abc', units=["-", "-"])
    json_pdtable = table_to_json_data(pdtable)
    pdtable_decoded = json_data_to_table(json_pdtable)  ### <-- raises a Value Error
```

**Comment by guilhermebs** [comment]
Possible fix: parse data using a `ParseFixer` before writing it and raise error if it fails

**Comment by guilhermebs** [comment]
Another idea is to add a "strict" kwarg in the Table constructor that checks if the data frame values are correct

**Comment by kevarding** [comment]
The root issue seems to be that you are even allowed to create the first `pdtable` right?
According to the specification units '-' are to be interpreted as dimensionless numeric values aka floats so column `b` is not allowed.

**Comment by guilhermebs** [comment]
That too, but we can't (and shouldn't) restrict how the users manipulate tables after they are created. They should be able to write invalid stuff in there.

**Comment by guilhermebs** [comment]
Closed with #124
